packages/core-components/src/lib/atoms/inputs/dropdown/Dropdown.svelte

```diff
@@ -21,15 +36,76 @@
 	/** @type {string} */
 	export let name;
 
+	export let multiple = false;
+
+	export let hideDuringPrint = true;
+
 	/** @type {string} */
-	export let defaultValue = null;
+	export let defaultValue = undefined;
```

Comment:
This appears now to not work properly - I can't unselect the defaultValue, as the dialog to select flashes and then closes immediately